Source/ProjectFirma.Web/Views/Agreement/DetailViewData.cs

```diff
@@ -84,6 +86,9 @@ public DetailViewData(Person currentPerson, Models.Agreement agreement, bool use
             List<AgreementGrantAllocation> agreementGrantAllocationsList = agreement.AgreementGrantAllocations.ToList();
             CurrentAgreementGrantAllocationsInSortedOrder = AgreementGrantAllocation.OrderAgreementGrantAllocationsByYearPrefixedGrantNumbersThenEverythingElse(agreementGrantAllocationsList);
             AgreementProjectsByGrantAllocations = ProjectAgreementByGrantAllocation.MakeAgreementProjectsByGrantAllocation(agreementGrantAllocationsList);

```

Comment:
Lets rip all the old stuff out too.
I want this whole function removed and no references to it: MakeAgreementProjectsByGrantAllocation()

Reply:
And the reverse one that exists